File: core java/src/Prime.java

class Prime {

	public static void main(String[] args) {
		  int n=14;
		    int i=2;
		    int flag=0;
		    while(i<=n){
		        if(n%i==0){
		             flag=1;
		        }
		        i++;
		        break;
		    }
		    if(flag==0){
		        System.out.println("prime");
		    }
		    else{
		       System.out.println("not prime");
		    }

	}

}

File: C programing/Assignment5/Ftype4/prime.c
//using type 4

#include<stdio.h>
int prime(int);
void main(){
    int n;
    printf("enter the number");
    scanf("%d",&n);
    int a=prime(n);
    if(a==1)
    printf("prime number");
    else
    printf("not prime number");
}
int prime(int n){
         int flag=0;
         int i=2;
         while(i<n){
            if(n%i==0){
                flag=1;
                break;
            }
            i++;
         }
         if(flag==0){
            return 1;
         }
         else{
            return 0;
         }
}